server/protx/data/api/views.py

```diff
@@ -159,18 +191,63 @@ def get_demographics_cached():
         return JsonResponse({"data": data, "meta": meta})
 
 
-@onboarded_required
-@ensure_csrf_cookie
+@ onboarded_required
+@ ensure_csrf_cookie
+def get_maltreatment(request):
+    return get_maltreatment_cached()
+
+
+@ memoize_db_results(db_file=maltreatment.db_name)
+def get_maltreatment_cached():
+    """Get maltreatment data
+
+    """
+    engine = create_engine(SQLALCHEMY_DATABASE_URL, connect_args={'check_same_thread': False})
+
+    with engine.connect() as connection:
+        result = connection.execute(MALTREATMENT_QUERY)
+        print(result)
+        # data = create_dict(result, level_keys=MALTREATMENT_JSON_STRUCTURE_KEYS)
+
+        # result = connection.execute(MALTREATMENT_MIN_MAX_QUERY)
+        # meta = create_dict(result, level_keys=MALTREATMENT_JSON_STRUCTURE_KEYS[:-1])
+        data = {}
+        meta = {}
+        return JsonResponse({"data": data, "meta": meta})
```

Comment:
these changes here and the change to `MALTREATMENT_QUERY` need to be reverted.  

we do the whole `create_dict` steps to put the response in a certain json form that we used at the start of the project (this allowed us to adopt the db from kelly but minimize our frontend changs).